src/main/java/de/dennisguse/opentracks/AbstractActivity.java

```diff
@@ -36,13 +38,31 @@ protected void onCreate(Bundle savedInstanceState) {
         setVolumeControlStream(TTSManager.AUDIO_STREAM);
 
         setContentView(getRootView());
+        this.ThemeUpdate();
     }
 
+   public void ThemeUpdate() {
+       int nightModeFlags = getResources().getConfiguration().uiMode & Configuration.UI_MODE_NIGHT_MASK;
```

Comment:
This logic can be moved into PreferenceUtils (see comment regarding global variable: OLEDFriendly).